easybuild/framework/easyconfig/tools.py

```diff
@@ -276,17 +276,21 @@ def get_paths_for(subdir=EASYCONFIGS_PKG_SUBDIR, robot_path=None):
 
 def alt_easyconfig_paths(tmpdir, tweaked_ecs=False, from_pr=False):
     """Obtain alternative paths for easyconfig files."""
-    # path where tweaked easyconfigs will be placed
-    tweaked_ecs_path = None
+
+    # paths where tweaked easyconfigs will be placed, easyconfigs listed on the command line take priority, tweaked
+    # dependencies are also created but these will only be appended to the robot path (and only used if strictly
+    # necessary)
+    tweaked_ecs_paths = None
     if tweaked_ecs:
-        tweaked_ecs_path = os.path.join(tmpdir, 'tweaked_easyconfigs')
+        tweaked_ecs_paths = {'tweaked_ecs_path': os.path.join(tmpdir, 'tweaked_easyconfigs'),
+                             'tweaked_ecs_deps_path': os.path.join(tmpdir, 'tweaked_dep_easyconfigs')}
```

Comment:
@ocaisa using a dictionary here is overkill imho, I think a 2-tuple is more appropriate here

```
tweaked_ecs_paths = (os.path.join(tmpdir, 'tweaked_easyconfigs'), os.path.join(tmpdir, 'tweaked_dep_easyconfigs'))
```